doc/source/plugins/b109_password_config_option_not_marked_secret.rst

```diff
@@ -1,5 +1,16 @@
-----------------------------------------------
-B109: password_config_option_not_marked_secret
-----------------------------------------------
+---------------------------------------------------------------
+B109: Test for a password based config option not marked secret
+---------------------------------------------------------------
 
-.. automodule:: bandit.plugins.secret_config_option
+This plugin has been deprecated.
```

Comment:
We should say removed, not deprecated.  Deprecated implies it still works and will be removed in the future.